s4e-backend/src/test/java/pl/cyfronet/s4e/data/repository/query/QueryBuilderTest.java

```diff
@@ -0,0 +1,67 @@
+package pl.cyfronet.s4e.data.repository.query;
+
+import lombok.extern.slf4j.Slf4j;
+import org.junit.jupiter.api.Test;
+import pl.cyfronet.s4e.BasicTest;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.is;
+
+@BasicTest
+@Slf4j
+public class QueryBuilderTest {
+    @Test
+    public void shouldReturnWholeQuery() {
+        QueryBuilder queryBuilder = new QueryEnding(
+                new QueryNumber(
+                        new QueryText(
+                                new QueryTime(
+                                        new QueryBuilderImpl()))));
```

Comment:
I'd autowire that

Reply:
Otherwise, make this test class not have @BasicTest annotation. It's pretty much unit test, which isn't bad in itself.